ls1intum/Artemis
# `General`: Change link target for Active Students plot header to course statistics

### Checklist
#### General


#### Client

### Motivation and Context


Resolves [#6496](https://github.com/ls1intum/Artemis/issues/6496)

### Description

Update the link for the small course statistics plot on the course management page to direct users to the detailed statistics page (/course-management/<ID>/course-statistics) instead of the general course management page (/course-management/ID). 

### Steps for Testing

Prerequisites:
- 1 Instructor/Editor/TA

1. Navigate to Course Management
2. Click on the title of the Active Students plot.
3. Confirm that you are redirected to the to the detailed statistics page (/course-management/<ID>/course-statistics).

### Review Progress






#### Performance Review

#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


![fix_plot_link](https://github.com/ls1intum/Artemis/assets/11020241/0d4ed36c-891f-4ef9-9670-98a5026454e1)
